### PR TITLE
update(Prison): Prison Break Require Cops

### PR DESCRIPTION
**Pull request by Savage912-Official**
Prison doors can now only be lockpicked if there are 4 cops on duty. This is to stop people from escaping prison when there is no cops on duty.

### CODE REVIEW

File: resources/usa_doormanager/cl_doormanager.lua [hunk 1]
```diff
@@ -308,30 +308,38 @@ AddEventHandler('doormanager:advancedPick', function()
...
-                TriggerEvent('usa:notify', 'You cannot use advanced lockpicks here!')
-                return
+              isMale = IsPedMale(playerPed)
+            end
+            if math.random() >= 0.40 then
+                TriggerServerEvent('911:LockpickingDoor', x, y, z, lastStreetNAME, isMale)
             end
+            TriggerEvent('lockpick:openlockpick')
+          else
+            TriggerEvent('usa:notify', 'Seems there isn\'t enough cops around!')
```
Comment (minipunch) [inline resources/usa_doormanager/cl_doormanager.lua:336]:
```suggestion
            TriggerEvent('usa:notify', 'Lockpick failed')
```

File: resources/usa_doormanager/sv_doormanager.lua [hunk 2]
```diff
@@ -433,3 +433,21 @@ end, {
     { name = "model", help = 'model of door in hash format'}
   }
 })
+
+-- Get Online Cops
+RegisterServerCallback {eventName = 'usa_doormanager:getCops', eventCallback = function(source)
```
Comment (minipunch) [inline resources/usa_doormanager/sv_doormanager.lua:438]:
```suggestion
RegisterServerCallback {eventName = 'usa_doormanager:getNumCops', eventCallback = function(source)
```

File: resources/usa_doormanager/cl_doormanager.lua [hunk 3]
```diff
@@ -308,30 +308,38 @@ AddEventHandler('doormanager:advancedPick', function()
...
-                if math.random() >= 0.40 then
-                    TriggerServerEvent('911:LockpickingDoor', x, y, z, lastStreetNAME, isMale)
-                end
-                TriggerEvent('lockpick:openlockpick')
+      local door = DOORS_TO_MANAGE[i]
+      local x, y, z = door.x, door.y, door.z
+      if Vdist(playerCoords, x, y, z) < 1.0 then
+        if door.advancedlockpickable then
+          local Cops = TriggerServerCallback {
+            eventName = 'usa_doormanager:getCops',
```
Comment (minipunch) [inline resources/usa_doormanager/cl_doormanager.lua:316]:
```suggestion
            eventName = 'usa_doormanager:getNumCops',
```

**Review by minipunch — CHANGES_REQUESTED** [review]
Would be ideal if u could make this only apply to cell block doors

Cause right now it's ALL advanced lockpick-able doors that require 4 cops with the way u did it right, which might not be what we want if we add more in the future

**Comment by Savage912-Official** [comment]
> Would be ideal if u could make this only apply to cell block doors
> 
> Cause right now it's ALL advanced lockpick-able doors that require 4 cops with the way u did it right, which might not be what we want if we add more in the future

Alright, ill work on it

**Comment by Savage912-Official** [comment]
Gonna come back to this later on no reason to have it open gonna make a new one when I have time for it.

**Pull request closed without merging**